### PR TITLE
Firmware auto updating

### PR DESCRIPTION
**Pull request by tannermpeterson**

### CODE REVIEW

File: components/playback/controls/player/DesktopPlayerControls.vue [hunk 1]
```diff
@@ -271,8 +332,43 @@ export default {
...
+      "firmware_update_available",
+      "firmware_update_dur_mins",
+    ]),
+    ...mapState("stimulation", ["stim_status"]),
+    ...mapGetters({
+      status_uuid: "flask/status_id",
+    }),
+    fw_update_available_labels: function () {
+      let duration = `${this.firmware_update_dur_mins} minute`;
+      if (duration !== 1) duration += "s";
```
Comment (luciipak) [inline components/playback/controls/player/DesktopPlayerControls.vue:349]: I don't think 'duration' would ever equal 1 in this case

File: components/playback/controls/player/DesktopPlayerControls.vue [hunk 2]
```diff
@@ -407,8 +515,18 @@ export default {
     },
     close_calibration_modal(idx) {
       this.$bvModal.hide("calibration-warning");
+      console.log("!!!");
```
Comment (luciipak) [inline components/playback/controls/player/DesktopPlayerControls.vue:518]: Console.log statements

File: components/playback/controls/player/DesktopPlayerControls.vue [hunk 3]
```diff
@@ -407,8 +515,18 @@ export default {
...
       if (idx === 1) this.$store.dispatch("playback/start_calibration");
     },
+    close_fw_update_available_modal(idx) {
+      this.$bvModal.hide("fw-update-available-message");
+      console.log("$$$");
```
Comment (luciipak) [inline components/playback/controls/player/DesktopPlayerControls.vue:523]: Console.log statements

File: components/playback/controls/player/DesktopPlayerControls.vue (hunk 1 above)
Comment (tannermpeterson) [reply components/playback/controls/player/DesktopPlayerControls.vue:349]: Yeah that's definitely a mistake, should be `this.firmware_update_dur_mins !== 1`

**Review by luciipak — APPROVED** [review]

**Pull request merged**